cmd/virt-chroot/tap-device-maker.go

```diff
@@ -20,6 +20,14 @@ func createTapDevice(name string, owner uint, group uint, queueNumber int) error
 		Group:      uint32(group),
 	}
 
+	// when netlink receives a request for a tap device with 1 queue, it uses
+	// the MULTI_QUEUE flag, which differs from libvirt; as such, we need to
+	// manually request the single queue flags, enabling libvirt can consume
```

Comment:
`enabling libvirt can consume the tap device` is probably a mixture of two comment versions.

Reply:
Right! Good catch.

Replaced the 'can' with 'to', which was what I originally intended to write.